chapters/two.md

```diff
@@ -20,6 +20,9 @@ React components are designed to be used just like any standard export in javasc
 how to export a component:
 
 ```js
+import React from "react";
+import ReactDom from "react-dom";
```

Comment:
This is not necessary please remove :P

Reply:
@leonp1991 do you wanna remove it so we can merge this